docs/features/server-side-editor-api.md

```diff
@@ -0,0 +1,347 @@
+---
+category: cloud-services
+order: 30
+meta-title: Cloud Services Server-side Editor API | CKEditor 5 Documentation
+meta-description: Learn how to use server-side API to manage content and collaboration data easily without running the editor.
+modified_at: 2025-06-05
+badges: [ premium ]
+---
+
+# Server-side editor API
+
+Server-side Editor API enables deep and complex integration of your application with all document data, enabling you to manipulate content and manage collaborative data such as suggestions, comments, and revision history, and much more, directly from your server-side code.
+
+The remote script REST API endpoint allows you to execute any JavaScript code that uses the CKEditor&nbsp;5 API, that could be executed by a browser, but without a need to open the editor by a human user. Instead, the script is executed on the Cloud Services server.
```

Comment:
```suggestion
The remote REST API endpoint allows you to execute any JavaScript code that uses the CKEditor&nbsp;5 API, that could be executed by a browser, but without a need to open the editor by a human user. Instead, the script is executed on the Cloud Services server.
```

Reply:
The idea here was that this is "remote script" as is "remotely executed script" (executed on the server). This will probably be changed to "The server-side editor REST API endpoint ..." anyway.